src/main/java/de/metas/ui/web/window/datatypes/json/DateTimeConverters.java

```diff
@@ -232,9 +231,9 @@ else if (valueObj instanceof CharSequence)
 			{
 				return null;
 			}
-			else if (isPossibleJdbcTimestamp(valueObj))
+			else if (isPossibleJdbcTimestamp(json))
 			{
-				final Timestamp timestamp = fromJdbcTimestampToLocalDateTime(valueObj);
+				final Timestamp timestamp = fromJdbcTimestampToLocalDateTime(json);
 				return fromObjectConverter.apply(timestamp);
 			}
```

Comment:
This code chunk is not handling the case when 
Basically, insead of lines 234:238, I would expect some like:

```java
 if (isPossibleJdbcTimestamp(json))
{
    try {
                final Timestamp timestamp = fromJdbcTimestampToLocalDateTime(json);
                 return fromObjectConverter.apply(timestamp);
    }
    catch(Exception e)
     {
           // log the exception including the `json` to be able to understand and the case later
           return fromJsonConverer.apply(json);
      }
}
```

Reply:
Added error handling